src/common/snippets/src/op/loop.cpp

```diff
@@ -37,180 +42,156 @@ std::shared_ptr<LoopEnd> LoopBegin::get_loop_end() const {
-    return loop_begin;
-}
-
-const std::vector<bool>& LoopEnd::get_is_incremented() const {
-    return m_is_incremented;
-}
-
-const std::vector<int64_t>& LoopEnd::get_element_type_sizes() const {
-    return m_element_type_sizes;
-}
-
-size_t LoopEnd::get_input_num() const {
-    return m_input_num;
-}
-
-size_t LoopEnd::get_output_num() const {
-    return m_output_num;
-}
-
-size_t LoopEnd::get_increment() const {
-    return m_work_amount_increment;
-}
-
-size_t LoopEnd::get_id() const {
-    return m_id;
-}
-
-void LoopEnd::set_is_incremented(std::vector<bool> is_incremented) {
-    OPENVINO_ASSERT(is_incremented.size() == m_input_num + m_output_num,
-                    "LoopEnd set_is_incremented is called with inconsistent is_incremented.size()");
-    m_is_incremented = std::move(is_incremented);
-}
-
-void LoopEnd::set_increment(size_t new_increment) {
-    m_work_amount_increment = new_increment;
-}
-
-void LoopEnd::set_id(size_t id) {
-    m_id = id;
-}
-
 void LoopEnd::validate_and_infer_types() {
     NODE_VALIDATION_CHECK(this, get_input_size() == 1, "LoopEnd must have one input");
     const auto loop_begin = ov::as_type_ptr<LoopBegin>(get_input_node_shared_ptr(0));
     const auto io_size = m_input_num + m_output_num;
     NODE_VALIDATION_CHECK(this, loop_begin != nullptr, "LoopEnd must have LoopBegin as the last argument");
-    NODE_VALIDATION_CHECK(this, m_is_incremented.empty() || m_is_incremented.size() == io_size,
-                          "is_incremented must be either empty or defined per every input & output of joined Loop. Expected size: ",
-                          io_size, " got ", m_is_incremented.size());
+
+#define VALIDATE_VALUES(values, name, default_value)                                                                         \
+    NODE_VALIDATION_CHECK(this, values.empty() || values.size() == io_size,                                                  \
+                          name, " must be either empty or defined per every input & output of joined Loop. Expected size: ", \
+                          io_size, " got ", values.size());                                                                  \
+    if (values.empty())                                                                                                      \
+        values.resize(io_size, default_value);
+
+    VALIDATE_VALUES(m_is_incremented, "is_incremented", true)
+    VALIDATE_VALUES(m_ptr_increments, "ptr_increments", 0)
+    VALIDATE_VALUES(m_finalization_offsets, "finalization_offsets", 0)
+    VALIDATE_VALUES(m_element_type_sizes, "element_type_sizes", 0)
+#undef VALIDATE_VALUES
+
     set_output_type(0, element::f32, ov::PartialShape{});
 }
 
 bool LoopEnd::visit_attributes(AttributeVisitor &visitor) {
     std::vector<int> int_incremented(m_is_incremented.cbegin(), m_is_incremented.cend());
+    auto work_amount = utils::value2str(m_work_amount);
```

Comment:
Now it looks so:
![image](https://github.com/openvinotoolkit/openvino/assets/43129309/92228162-690e-46af-a900-29b8512da0c9)